### PR TITLE
Release 0.1.1

### LINKED ISSUE #6: Change in Ruby base64 encryptor
When updating the gem to ruby 2.4 or using it on a project using ruby 2.4, we'll face the same issue as below.
https://github.com/attr-encrypted/encryptor/issues/26

### PR DESCRIPTION
**Pull request by bengreville**
Patch release allows for Ruby 2.4 and higher OpenSSL support, which enforces a 32 byte string length encryption key. 

Existing applications using this gem that are encrypting tokens using the correct length string can continue running the older versions.

This patch also removes a deprecation warning described here: https://github.com/attr-encrypted/attr_encrypted/issues/264

Fixes: https://github.com/NuffieProductions/ClaimToken/issues/6

### CODE REVIEW

File: lib/claim_token/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module ClaimToken
-  VERSION = "0.1.0"
+  VERSION = "0.1.1".freeze
```
Comment (vprigent) [inline lib/claim_token/version.rb]: Do we actually need to specify freeze now ? I though this was being automatically done by ruby ?

Comment (bengreville) [reply lib/claim_token/version.rb]: According to this stack overflow discussion you should do this or add the frozen string literal magic comment until Ruby 3, in which it'd do this by default https://stackoverflow.com/a/37799399

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -8,16 +8,16 @@
...
+    allow(OpenSSL::Cipher).to receive(:new).and_return(cipher)
   end
 end
 
 def use_test_configuration
   before do
     ClaimToken.configure do |config|
-      config.shared_encryption_key = "uFeb25D_z0BAGhgH7WKy8QBSat6kxfDa6PrKw1ox69U="
+      config.shared_encryption_key = "fba0d989f259ad36ada12c127356d420"
       config.digest_secret = "d22b90c9172649eaf49cd185f73bd4a53ec3ff4dc6f7d51f9dbaac62421e6dd297b38efcd431a7f2"
```
Comment (vprigent) [inline spec/spec_helper.rb]: This key is still 64 bits long, no potential issue there ?

**Review by vprigent — APPROVED** [review]

File: claim_token.gemspec [hunk 3]
```diff
@@ -6,7 +6,7 @@ require 'claim_token/version'
 Gem::Specification.new do |spec|
   spec.name          = "claim_token"
   spec.version       = ClaimToken::VERSION
-  spec.authors       = ["CricHQ"]
+  spec.authors       = ["Daniel Zollinger", "Ben Greville"]
```
Comment (dznz) [inline claim_token.gemspec:9]: Aww this lil library still getting love 😁

**Pull request merged**